nixos/modules/hardware/video/nvidia.nix

```diff
@@ -139,9 +142,54 @@ in
       description = ''
         Enable render offload support using the NVIDIA proprietary driver via PRIME.
 
-        If this is enabled, then the bus IDs of the NVIDIA and Intel GPUs have to be
-        specified (<option>hardware.nvidia.prime.nvidiaBusId</option> and
-        <option>hardware.nvidia.prime.intelBusId</option>).
+        If this is enabled, then the bus IDs of the NVIDIA and Intel/AMD GPUs have to
+        be specified (<option>hardware.nvidia.prime.nvidiaBusId</option> and
+        <option>hardware.nvidia.prime.intelBusId</option> or
+        <option>hardware.nvidia.prime.amdgpuBusId</option>).
+      '';
+    };
+
+    hardware.nvidia.prime.offload.enableOffloadCmd = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Adds a `nvidia-offload` convenience script to <option>environment.systemPackages</option>
+        for offloading programs to an nvidia device. To work, should have also enabled
+        <option>hardware.nvidia.prime.offload.enable</option> or <option>hardware.nvidia.prime.reverse_sync.enable</option>
+
+        Example usage `nvidia-offload sauerbraten_client`
+      '';
+    };
+
+    hardware.nvidia.prime.reverse_sync.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Warning: This feature is relatively new, depending on your system this might
+        work poorly. AMD support, especially so.
+        See: https://forums.developer.nvidia.com/t/the-all-new-outputsink-feature-aka-reverse-prime/129828
+
```

Comment:
I am not sure about this warning. We generally end up never updating those module descriptions, so I would remove this because the tendency is that eventually NVIDIA will get those drivers write (at least, I hope so).

This is better documented in our Wiki.